src/test/java/com/ge/predix/integration/cloudfoundry/CFClientTest.java

```diff
@@ -0,0 +1,140 @@
+		this.headers.put(HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON_VALUE);
+	}
+	
+	public String testCreateServiceInstance(String uaaUrl) throws Exception {
+
+		String serviceInstanceName = "dcs-metering-test";
+		//String trustedIssuerId = "https://e63b2f29-44a4-4e12-a444-a5677beabe5f.predix-uaa.run.aws-usw02-dev.ice.predix.io/oauth/token";
+
+		String serviceInstanceGuid = createNewServiceInstance(serviceInstanceName, uaaUrl);
+		try {
+			verifyServiceInstanceCreated(serviceInstanceGuid);
+			
+			System.out.println("Inside testCreateServiceInstance ********");			
+		} catch (Exception e) {
+			Assert.fail(e.toString());
+		} 
+		return serviceInstanceGuid;
+	}
+	
+	private String createNewServiceInstance(final String serviceInstanceName, final String trustedIssuerId)
+			throws Exception {
+
+		HashMap<String, Object> serviceRequest = new HashMap<String, Object>();
+		serviceRequest.put("space_guid", this.cfSpaceGuid);
+		serviceRequest.put("name", serviceInstanceName);
+		serviceRequest.put("service_plan_guid", this.acsServicePlanGuid);
+		Map<String, List<String>> issuerMap = new HashMap<String, List<String>>();
+		issuerMap.put("trustedIssuerIds", Arrays.asList(trustedIssuerId));
+		serviceRequest.put("parameters", issuerMap);
+
+		URI createInstanceURI = URI.create(this.cfControllerURL + "/v2/service_instances");
+		String response = this.cfRestTemplate.postForObject(createInstanceURI, serviceRequest, String.class);
+
+		@SuppressWarnings("unchecked")
+		Map<String, Object> responseMap = new ObjectMapper().readValue(response, Map.class);
+		@SuppressWarnings("unchecked")
+		Map<String, Object> metadataMap = (Map<String, Object>) responseMap.get("metadata");
+		return (String) metadataMap.get("guid");
+	}
+	
+    public UaaInstance setUAAInstance(String secret, String serviceInstanceName) throws JsonParseException, JsonMappingException, IOException {
+		HashMap<String, Object> serviceRequest = new HashMap<String, Object>();
+		serviceRequest.put("space_guid", this.cfSpaceGuid);
+		serviceRequest.put("name", serviceInstanceName);
+		serviceRequest.put("service_plan_guid", this.uaaServicePlanGuid);
+		
+		Map<String, String> issuerMap = new HashMap<String, String>();
+		issuerMap.put("adminClientSecret", secret);
+		serviceRequest.put("parameters", issuerMap);
+		
+		URI createInstanceURI = URI.create(this.cfControllerURL + "/v2/service_instances");
+		String response = this.cfRestTemplate.postForObject(createInstanceURI, serviceRequest, String.class);
+
+		@SuppressWarnings("unchecked")
+		Map<String, Object> responseMap = new ObjectMapper().readValue(response, Map.class);
+		@SuppressWarnings("unchecked")
+		Map<String, Object> metadataMap = (Map<String, Object>) responseMap.get("metadata");
+		String uaaGuid = (String)metadataMap.get("guid");
+		StringBuilder uaaUrl = new StringBuilder().append("https://").append(uaaGuid).append(".predix-uaa.run.aws-usw02-dev.ice.predix.io/oauth/token");
```

Comment:
Make predix-uaa.run.aws-usw02-dev.ice.predix.io/oauth/token as a environment variable, instead of hardcoding it